core/test/unit/web/middleware/uncapitalise_spec.js

```diff
@@ -123,24 +123,21 @@ describe('Middleware: uncapitalise', function () {
             done();
         });
     });
-
+    
```

Comment:
I've noticed you have these stray empty space symbols left in few places. One great config change you could make in the editor you are using is enabling "Trim Trailing Whitespace" (at least that's how it's called in VS Code) :wink:

Reply:
I have enabled this setting under visual studio code but it's not working.
Researching into next steps.

Reply:
so after reviewing this more, under .editorconfig file
```
[*]
insert_final_newline = true
```

is adding in the extra new line is it not?